### PR TITLE
Bump to Nette 3.0

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "pixidos/gpwebpay",
+  "name": "tomas-votruba/gpwebpay",
```
Comment (ondraondra81) [inline composer.json:2]: Why?

File: src/Pixidos/GPWebPay/Components/GPWebPayControl.php [hunk 2]
```diff
@@ -62,8 +62,6 @@ class GPWebPayControl extends UI\Control
      */
     public function __construct(IOperation $operation, IProvider $provider, IContainer $control = null, $name = null)
     {
-        parent::__construct($control, $name);
```
Comment (ondraondra81) [inline src/Pixidos/GPWebPay/Components/GPWebPayControl.php:65]: when you remove this, why not you does not remove `IContainer $control = null, $name = null` from constructor?

**Review by ondraondra81 — COMMENTED** [review]
Thx you. But can you fix the tests pls.

**Comment by TomasVotruba** [comment]
Oh, it's not a PR, it's just temp diff to see the difference.
We used it as hotfix fork at training to make app work.

Sorry, I thought this package is dead

**Pull request closed without merging**